src/server/services/VideoresServices.js

```diff
@@ -20,6 +20,21 @@ class VideoresServices {
     }
     return docs;
   }
+
+  static async formatTermVidRes(media) {
+    for (const courseMedia of media) {
+      for (const entry of courseMedia.listings) {
+        delete entry._id;
```

Comment:
I'm a little confused of why we need to delete _id here.
Just to make sure, for the video res displayed to students, the _id of those video reserve medias are not deleted right? I use those _id as playerId of jwplayers.

Reply:
@maxpluez Yeah, this doesn't modify anything in the database, but I'll probably refactor the db aggregation instead to only include the fields needed for the frontend anyway (here, I initially wanted just to delete certain fields that I knew for sure wouldn't be needed on the frontend)